Meter.c

```diff
@@ -345,8 +344,8 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
    }
    for (; i < nValues - 1; i += 2, k++) {
       int pix = GraphMeterMode_pixPerRow * GRAPH_HEIGHT;
-      int v1 = CLAMP((int) lround(data->values[i] * pix), 1, pix);
-      int v2 = CLAMP((int) lround(data->values[i + 1] * pix), 1, pix);
+      int v1 = CLAMP((int) lround(data->values[i] / this->total * pix), 1, pix);
```

Comment:
This should probably get protection against division-by-zero as meters update "total" dynamically.
Of course meters should not set a total to zero but rounding happens and better safe than sorry.
Same for the `v2` line below.